### PR TITLE
Adding polls project for code review.

### PR DESCRIPTION
**Pull request by dagosi**
This pull request contains all the poll project that is mentioned in the django home page (djangoproject.com) as an example project.

### CODE REVIEW

File: mysite/polls/#models.py# [hunk 1]
```diff
@@ -0,0 +1,28 @@
```
Comment (scastillo) [inline mysite/polls/#models.py#]: Yea this is other .gitignore candidate

File: mysite/polls/tests.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
```
Comment (scastillo) [inline mysite/polls/tests.py]: What kind of test you think we can add here? try running `./manage.py test` first and see if this example test is working. Then maybe you can make a test that checks if the poll is being displayed to the user or something. I will be adding more to the README on tests tomorrow.

File: mysite/polls/urls.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from django.conf.urls import patterns, include, url
+from django.views.generic import DetailView, ListView
+from polls.models import Poll
+
+urlpatterns = patterns('',
+    url(r'^$',
+        ListView.as_view(
+            queryset=Poll.objects.order_by('-pub_date')[:5],
```
Comment (scastillo) [inline mysite/polls/urls.py]: Very nice. generic views rocks!

**Comment by scastillo** [comment]
Awesome, now deploy it to heroku so we can play a little :) and remember to use the odesk tool whenever you are working on this

**Comment by dagosi** [comment]
The app is now deployed on Heroku. You can see the polls created at http://bildung.herokuapp.com/polls/ and you can create the polls in http://bildung.herokuapp.com/admin/ with the follow information:

User: bildung
pass: bildung

**Pull request closed without merging**